src/Form/_variables.scss

```diff
@@ -0,0 +1,263 @@
+
+// Buttons + Forms
+//
+// Shared variables that are reassigned to `$input-` and `$btn-` specific variables.
```

Comment:
I don't actually see any `$btn-*` variables in this folder. Are they needed here, or do we just need the `$input-btn-*` styles?

Reply:
These are the common, shared variables between buttons and forms. Like in a comment above, these variables are defined in both places right now. The definitions in buttons technically have priority because they're included first. I'll pull these back out the `core` directory.